pmm-app/codeceptConfigHelper.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   pageObjects: {
     I: './tests/custom_steps.js',
+    codeceptjsConfig: './pr.codecept.js',
```

Comment:
what is this for?

Reply:
okay I see

Reply:
to simplify using it. like [here](https://github.com/percona/grafana-dashboards/blob/8b4b2d80b5c9d27ed104c9888079771666dae201/pmm-app/tests/pages/api/settingsAPI.js#L5)